docs/components/optimize/userguide/process-analysis/report-analysis/define-reports.md

```diff
@@ -0,0 +1,113 @@
+---
+id: define-reports
+title: Define reports
+description: In this section of the report builder, you are characterizing the output of the report.
+---
+
+In this section of the report builder, you are characterizing the output of the report. Basically, you are defining "I want to view ... grouped by ... visualized as ...". To understand better what "View" and "Group by" mean, you can use the analogy of a graph, where "View" is the y-axis and "Group by" is the x-axis.
+
+First, you need to select which part of the data you want to view. Optimize differentiates between the view (e.g. process instance or flow node) and the measure (e.g. count or duration):
+
+1. Raw Data: View just a table with the actual data listed as rows. This can come in handy if you found interesting insights in certain process instances and need detailed information about those instances, or you are exploring a process definition with a limited number of instances. This report type also allows you to inspect raw [object variable values](../../../../../self-managed/optimize-deployment/configuration/object-variables.md).
+2. Process instance
```

Comment:
```suggestion

2. Process instance
```

Reply:
Is this an intentional change to add a line or typo?

Reply:
intentional as the other list items are also preceded with an empty line